packages/toolkit/src/shared/cloudformation/cloudformation.ts

```diff
@@ -379,13 +379,13 @@ export interface TemplateResources {
 export function isValidFilename(filename: string | vscode.Uri): boolean {
     filename = typeof filename === 'string' ? filename : filename.fsPath
     filename = filename.trim()
-    if (!filename.match(templateFileRegexPattern)) {
+    if (!templateFileRegexPattern.test(filename)) {
         return false
     }
     // Note: intentionally _not_ checking `templateFileExcludePattern` here, because while excluding
     // template files in .aws-sam/ is relevant for the workspace scan, it's irrelevant if such
     // a file was opened explicitly by the user.
-    return !filename.match(devfileExcludePattern)
+    return !devfileExcludePattern.test(filename)
```

Comment:
This is only a stylistic change. `test()` returns a boolean where as `match()` returns a list of matches. So `test()` better fits the intention.